spaceship/spec/tunes/app_submission_spec.rb

```diff
@@ -22,7 +22,7 @@
       submission.content_rights_contains_third_party_content = true
       submission.content_rights_has_rights = true
       submission.add_id_info_uses_idfa = false
-      submission.complete!
+      submission.complete!(false)
```

Comment:
Is there a way to set `false` as the default of no param is supplied to this method? Would be much neater that way.

Reply:
Good idea. Added that.